services/github/github-manifest.service.js

```diff
@@ -0,0 +1,159 @@
+'use strict'
+
+const Joi = require('joi')
+const { renderVersionBadge } = require('../../lib/version')
+const {
+  individualValueSchema,
+  transformAndValidate,
+  renderDynamicBadge,
+} = require('../dynamic-common')
+const { ConditionalGithubAuthService } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+
+const schema = Joi.object({
+  version: individualValueSchema,
+}).required()
+
+class GithubManifestVersion extends ConditionalGithubAuthService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'github/manifest-json/v',
+      format: '([^/]+)/([^/]+)/?([^/]+)?',
+      capture: ['user', 'repo', 'branch'],
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'GitHub manifest version',
+        pattern: 'manifest-json/v/:user/:repo',
+        namedParams: {
+          user: 'RedSparr0w',
+          repo: 'IndieGala-Helper',
+        },
+        staticPreview: this.render({ version: 2 }),
+        documentation,
+      },
+      {
+        title: 'GitHub manifest version',
+        pattern: 'manifest-json/v/:user/:repo/:branch',
+        namedParams: {
+          user: 'RedSparr0w',
+          repo: 'IndieGala-Helper',
+          branch: 'master',
+        },
+        staticPreview: this.render({ version: 2, branch: 'master' }),
+        documentation,
+      },
+    ]
+  }
+
+  static render({ version, branch }) {
+    return renderVersionBadge({
+      version,
+      tag: branch,
+      defaultLabel: 'manifest',
+    })
+  }
+
+  async handle({ kind, which, user, repo, branch }) {
```

Comment:
It's not clear to me where `kind` and `which` are coming from, nor if/where they are used.

Reply:
Removed, thanks!